hudi-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadata.java

```diff
@@ -92,6 +93,7 @@ public class HoodieBackedTableMetadata extends BaseTableMetadata {
   private HoodieTableConfig metadataTableConfig;
   // should we reuse the open file handles, across calls
   private final boolean reuse;
+  private final boolean cacheFileListing;
```

Comment:
I assume this flag will be set as a session prop in Presto. But, where are we using this flag in metadata fs view?

Reply:
Changes are rolled back.  I instead override the `reuse` flag as mentioned.